src/util/rocksdb/event_listener.rs

```diff
@@ -117,8 +117,8 @@ impl rocksdb::EventListener for CompactionListener {
             }
         }
 
-        let mut input_files = HashSet::default();
-        let mut output_files = HashSet::default();
+        let mut input_files = HashSet::with_capacity(info.input_file_count());
```

Comment:
How about `HashSet::with_capacity_and_hasher`?

Reply:
The numbers of output_files and input_files are not too large, there is no performance issue.